src/Module.Client/GUI/CrpgMissionMarkerVM.cs

```diff
@@ -237,6 +251,16 @@ private void OnTeamChanged(NetworkCommunicator peer, Team previousTeam, Team new
         }
     }
 
+    private void OnCommanderUpdated(BattleSideEnum side)
+    {
+        BattleSideEnum myPeerSide = GameNetwork.MyPeer.ControlledAgent?.Team?.Side ?? BattleSideEnum.None;
```

Comment:
Taleworld code hint at mypeer sometimes bein null. add a ? just in case
![image](https://github.com/namidaka/crpg/assets/47090987/80946d36-7559-49fd-b166-0553c5927c8f)

Reply:
added null checks for MyPeer